website/docs/cloud-docs/api-docs/policy-checks.mdx

```diff
@@ -250,3 +250,239 @@ The GET endpoints above can optionally return related resources, if requested wi
 | ------------------ | -------------------------------------- |
 | `run`              | The run this policy check belongs to.  |
 | `run.workspace`    | The associated workspace of the run.   |
+
+<!-- BEGIN: TFC:only name:opa-policies -->
+
+## List Policy Evaluations in the Task Stage
+
+This endpoint is only available for OPA policies.
```

Comment:
@mrinalirao I don't know what the "Task Stage" is. It doesn't appear as one of the states on this page....https://www.terraform.io/cloud-docs/run/states